docs/user-guide/timepix_images.ipynb

```diff
@@ -204,15 +204,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "sample_2d = sample_img.sum('wavelength').plot(\n",
```

Comment:
What is the resoning behind switching to computing the mean instead of the sum?

Reply:
It was requested by Robin and Christian. They need `average` counts to fit the model...?

Reply:
Let's talk about this over a call tomorrow. Not sure I understood what you meant.